vars/gatlingExecuteTests.groovy

```diff
@@ -0,0 +1,100 @@
+import com.sap.piper.ConfigurationHelper
+import com.sap.piper.Utils
+
+import static com.sap.piper.Prerequisites.checkScript
+
+import com.sap.piper.GenerateDocumentation
+import groovy.transform.Field
+
+@Field String STEP_NAME = getClass().getName()
+
+@Field Set GENERAL_CONFIG_KEYS = []
+
+@Field Set STEP_CONFIG_KEYS = [
+    /**
+     * Defines the behavior in case tests fail. When this is set to `true` test results cannot be recorded using the `publishTestResults` step afterwards.
+     * @possibleValues `true`, `false`
+     */
+    'failOnError',
+    /**
+     * Path to the pom.xml file containing the performance test Maven module, for example `performance-tests/pom.xml`.
+     */
+    'testModule',
+    /**
+     * Optional List of app URLs and corresponding Jenkins credential IDs.
+     */
+    'appUrls'
+]
```

Comment:
I would expect that you would also need some stashing to use it in the general purpose pipeline

Reply:
Is this addressed by specifying the stashContent parameter in the defaults?

Reply:
Don't think so. I thought for groovy steps you would have to implement it. Or did I miss the place where this default value is read?